manimlib/extract_scene.py

```diff
@@ -73,6 +74,11 @@ def get_scenes_to_render(scene_classes, scene_config, config):
         found = False
         for scene_class in scene_classes:
             if scene_class.__name__ == scene_name:
+                if hasattr(scene_class, "CONFIG"):
+                    scene_class_config = scene_class.CONFIG
+                else:
+                    scene_class_config = {}
+                scene_config = merge_dicts_recursively(scene_config, scene_class_config)
```

Comment:
This will override the CLI flags like `-c` (`--color`)

Reply:
> This will override the CLI flags like `-c` (`--color`)

That should be the intended behavior though? I mean it wouldn't make sense to use both `CONFIG` and CLI flag to change the background color of a given scene. One or the other has to be overridden. And as you mentioned below, this PR goes with the priority order `Scene.CONFIG > CLI flags`

Reply:
Yes, we rarely use this, but it does not mean that we should not support such use.
Sometimes, you may try to change some other colors to render different results. At this time, it is very convenient to use CLI flags, but if you happen to set the color in `CONFIG` at this time, it will completely fix the background color.
Therefore, if there is a conflict between CLI flags and `CONFIG`, the CLI flags which is closer to the call should be kept.
In addition, the old version in `cairo-backend` supports this (3>2>1 as I said below)

Reply:
Yeah, got it. I didn't know that the `cairo-backend` supported 3>2>1. I'll try to get this order working with this PR, thanks for the review!